configuration/configuration.go

```diff
@@ -1123,7 +1123,7 @@ func (s *SectionParser) httpCheck() interface{} {
 			hc.ExclamationMark = h.ExclamationMark
 			hc.Match = h.Match
 			hc.Pattern = h.Pattern
-			*hc.Type = h.Type
+			hc.Type = &h.Type
```

Comment:
Definitely, this is the issue creating nil pointer dereference, but linting is complaining about this.

```
```
configuration/configuration.go:1126:15: exporting a pointer for the loop variable h (exportloopref)
                        hc.Type = &h.Type
```

I think we can solve it using the `misc.StringP` utility func: could you give it a try?

Reply:
done